crates/ironrdp-cliprdr-native/src/windows/mod.rs

```diff
@@ -64,7 +65,7 @@ impl core::fmt::Display for WinCliprdrError {
             WinCliprdrError::Alloc => write!(f, "failed to allocate global memory"),
             WinCliprdrError::DataReceiveTimeout => write!(f, "failed to receive data from remote clipboard"),
             WinCliprdrError::RenderFormat => write!(f, "failed to render clipboard format"),
-            WinCliprdrError::WinAPI(_error) => write!(f, "WinAPI error"),
+            WinCliprdrError::WinAPI(error) => write!(f, "WinAPI error: {error}"),
```

Comment:
issue: This is intentional; the source error is already propagated via the `source()` method in `core::error::Error` implementation below

```suggestion
            WinCliprdrError::WinAPI(_error) => write!(f, "WinAPI error"),
```